keel-core/src/main/kotlin/com/netflix/spinnaker/keel/lifecycle/LifecycleEvent.kt

```diff
@@ -0,0 +1,28 @@
+package com.netflix.spinnaker.keel.lifecycle
+
+import java.time.Instant
+
+data class LifecycleEvent(
```

Comment:
can `LifecycleEvent` and `LifecycleStep` live in the same place?

Reply:
I think it's best practice to keep each thing to its own class file